src/Illuminate/Database/Schema/SQLiteBuilder.php

```diff
@@ -34,17 +34,21 @@ public function dropDatabaseIfExists($name)
     /**
      * Get the tables for the database.
      *
+     * @param  bool  $withSize
      * @return array
      */
-    public function getTables()
+    public function getTables($withSize = true)
     {
-        $withSize = false;
-
-        try {
-            $withSize = $this->connection->scalar($this->grammar->compileDbstatExists());
-        } catch (QueryException $e) {
-            //
-        }
+        $withSize = match ($withSize) {
+            false => false,
+            default => function () {
```

Comment:
You are returning a function here? match doesn't work like that.

Reply:
@hafezdivandari Yes you're right.